coursier.bzl

```diff
@@ -745,23 +774,23 @@ def _coursier_fetch_impl(repository_ctx):
         override_targets = repository_ctx.attr.override_targets,
     )
 
+    # This repository rule can be either in the pinned or unpinned state, depending on when
```

Comment:
Why is this here? This is the implementation of the pinned repo, so the repository name should never have `unpinned_`?

Reply:
This was already here, I just moved it up so i can use it in BUILD.  I didn't check if it was needed in practice.

Reply:
This is `_coursier_fetch_impl` which is called with `name = name if maven_install_json == None else "unpinned_" + name,` -- so the check is required afaict.